### ISSUE TITLE
tracking and segmentation params

### ISSUE THREAD

**Issue by 36000** [yeatmanlab/pyAFQ#1032]
Should we get rid of these distinctions? It seems to just confuse people. IE, segmentation and tracking params are passed in like any other parameter, not in a separate dictionary.

**Comment by arokem** [comment]
Yeah - I think it would be clearer. I think that we wanted to avoid having a lot of different key-word arguments into the API, but this ends up as another level that users need to keep track of, without actually changing the number of different names and values that you'd need to keep track of.
